configs/default/IFRC-IFLIGHT_F411_AIO32.config

```diff
@@ -0,0 +1,110 @@
+# Betaflight / STM32F411 (S411) 4.0.0 NOV_28_2020_BY_RONG
+
+board_name IFLIGHT_F411_AIO32
+manufacturer_id IFRC
+
+# resources
+resource BEEPER 1 B02
+resource MOTOR 1 B04
+resource MOTOR 2 B05
+resource MOTOR 3 B06
+resource MOTOR 4 B07
+resource PPM 1 A03
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 11 B03
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 11 B10
+resource SERIAL_RX 12 A00
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C13
+resource LED 2 C14
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource ADC_BATT 1 B00
+resource ADC_CURR 1 B01
+resource FLASH_CS 1 A15
+resource OSD_CS 1 B12
```

Comment:
Looks like now you have removed _all_ of the features - from the look of it this board does have an OSD chip, so `feature OSD` should be set to have this enabled by default (as per https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md#26-add-custom-settings-for-your-board-to-the-unified-target-configuration).

Reply:
Okay. I will change it.